server/middlewares/validators/videos/videos.ts

```diff
@@ -198,6 +204,21 @@ const videosCustomGetValidator = (
 const videosGetValidator = videosCustomGetValidator('all')
 const videosDownloadValidator = videosCustomGetValidator('all', true)
 
+const videoFileMetadataGetValidator = getCommonVideoEditAttributes().concat([
+  param('id').custom(isIdOrUUIDValid).not().isEmpty().withMessage('Should have a valid id'),
+  param('videoFileId').custom(isIdValid).not().isEmpty().withMessage('Should have a valid videoFileId'),
+
+  async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+    logger.debug('Checking videoFileMetadataGet parameters', { parameters: req.params })
+
+    if (areValidationErrors(req, res)) return
+    if (!await doesVideoExist(req.params.id, res)) return
```

Comment:
I don't think we need this.

Instead, transform `doesVideoExistForVideoFile`  to `doesVideoFileOfVideoExist` and fetch `videoFile` according to an `id` and `videoId`. Just fetch `videoFile`, as you don't need `video`.

Also, instead of using `toInt` here you can use it line 209: `isEmpty().toInt()`

Reply:
I still need to check for the video, since it gives me an `id` of the video whereas without it I just have its `uuid`.

Reply:
Ah, figured it out. nevermind.